### PR TITLE
Add definition to content

### PR DESCRIPTION
**Pull request by LJBabbage**
### What is the context of this PR?
Describe what you have changed and why, link to other PRs or Issues as appropriate.

### How to review 
Describe the steps required to test the changes (include screenshots if appropriate).

### CODE REVIEW

File: eq_translations/extractable_strings.py [hunk 1]
```diff
@@ -65,6 +65,16 @@
...
+    {
+        "json_path": "$..content.contents[*].definition.title",
+        "description": "Definition title",
+        "context": "Content",
+    },
+    {
+        "json_path": "$..content.contents[*].definition.contents[*].description",
+        "description": "Definition description",
+        "context": "Content",
+    },
```
Comment (MebinAbraham) [inline eq_translations/extractable_strings.py:77]: Examples of these should be added to the test schema with the correct extraction and translation files.

Comment (LJBabbage) [reply eq_translations/extractable_strings.py:77]: We are really erratic at doing that

Comment (MebinAbraham) [reply eq_translations/extractable_strings.py:77]: We missed like 1 or 2, but we do them if we don't forgot. Even if there is no automated test, it highlights the extraction of the pot and transactions but the package is released. There is 1 test that extracts all translate items from the test schema and asserts that they are of the correct type.

Comment (LJBabbage) [reply eq_translations/extractable_strings.py:77]: 1 or 2 is being generous, there isn't a single content block in test_languages.json and very few block types

Comment (MebinAbraham) [reply eq_translations/extractable_strings.py:77]: We used to have a schema to have all extractable strings. We must have removed it. Maybe we just need a tidy up card for this remove so 👍

**Review by MebinAbraham — APPROVED** [review]

**Review by JamesGardiner — APPROVED** [review]

**Pull request merged**